bazel/scripts/test_python_service_imports.sh

```diff
@@ -37,7 +37,7 @@ collect_services() {
         SERVICES=( "${SERVICE_PATH}" )
     else
         echo "Multiple services specified:"
-        mapfile -t SERVICES < <(bazel query "attr(main, main.py, kind(py_binary, //${SERVICE_PATH}...))")
+        mapfile -t SERVICES < <(bazel query "attr(tags, service, kind(py_binary, //${SERVICE_PATH}...))")
```

Comment:
I tested this for `//...` and the result is the same with and without change.

Reply:
Which is good, right?

Reply:
Yes :+1: